components/Nav/Navbar/index.tsx

```diff
@@ -330,14 +330,15 @@ const NavBarContent: React.FC = styled(({ className }) => {
             <DropdownLogo />
             <ul>
                 <li className={linkStyles + (route === 'trade' ? ' selected' : '')}>
-                    <span className="trade-toggle">
-                        <Link href="/trade/basic">
-                            <div className={`${secondaryRoute === 'basic' ? 'selected' : ''}`}>Basic</div>
-                        </Link>
-                        <Link href="/trade/advanced">
-                            <div className={`${secondaryRoute === 'advanced' ? 'selected' : ''}`}>Advanced</div>
-                        </Link>
-                    </span>
+                    {/*TODO: Add back basic/advanced trading toggle*/}
+                    {/*<span className="trade-toggle">*/}
+                    {/*    <Link href="/trade/basic">*/}
+                    {/*        <div className={`${secondaryRoute === 'basic' ? 'selected' : ''}`}>Basic</div>*/}
+                    {/*    </Link>*/}
+                    {/*    <Link href="/trade/advanced">*/}
+                    {/*        <div className={`${secondaryRoute === 'advanced' ? 'selected' : ''}`}>Advanced</div>*/}
+                    {/*    </Link>*/}
+                    {/*</span>*/}
                     <Link href="/trade/basic">
```

Comment:
This goes to basic
```suggestion
                    <Link href="/trade/advanced">
```

Reply:
remember to try and test everything

Reply:
It works because of `'/trade/basic': { page: '/trade/[interface]' },`. It's just that '/basic' might be redundant since there's just one trading interface now. Updating now.